content/src/controller/Controller.ts

```diff
@@ -351,17 +351,45 @@ export class Controller {
             enumFields.push(DeploymentField.AUDIT_INFO)
         }
 
+        const sortFieldEnum: SortingField | undefined | 'unknown' = this.asEnumValue(SortingField, sortingField)
```

Comment:
The variables `sortingField` and `sortingOrder` could be avoided. We could just do `this.asEnumValue` when reading the query params, like all other params being read